scripts/sync-bench/bn_query_importer.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Usage: ./bn_query_node
+
+polling_interval=5
+
+importer_request="curl -s -H 'Content-Type: application/json' -X GET http://localhost:8200/api/stats/blocksCount"
```

Comment:
I would use https://stedolan.github.io/jq/ to parse Json, something like

`curl -s -H 'Content-Type: application/json' -X GET http://localhost:8200/api/stats/blocksCount | jq .Right`